grails-app/domain/payer/Payer.groovy

```diff
@@ -0,0 +1,18 @@
+package payer
+
+import cobra.BaseDomain
+
+class Payer extends BaseDomain{
+
+    String name
+    String email
+    String cpfCnpj
+
+    static mapping = {
+        cpfCnpj unique: true, length: 14
+    }
+    static constraints = {
+        name blank: false
+        email blank: false, email: true
```

Comment:
O que seria o `email: true`?

Reply:
Na documentação diz "Validates that a String value is a valid email address."